axelrod/tests/strategies/test_gobymajority.py

```diff
@@ -74,10 +74,15 @@ def test_strategy(self):
         self.responses_test([D], [C, D, D, D], [D, D, C, C])
         self.responses_test([D], [C, C, D, D, C], [D, D, C, C, D])
 
-        # Test tie break rule for soft=True
-        player = self.player(soft=True)
-        opponent = axelrod.Cooperator()
-        self.assertEqual('C', player.strategy(opponent))
+    def test_soft(self):
+        pass
```

Comment:
👍  Could you add a comment to explain the need for line 78? (I had to ask this again to keep github's review process happy).